openvpn/tunnel/setup_tun_linux.go

```diff
@@ -90,9 +90,13 @@ func (service *LinuxTunDeviceManager) createTunDevice(device tunDevice) (err err
 		return err
 	}
 
-	used, err := service.deviceUsed(device)
-	if err != nil {
-		return
+	var used bool
+
+	if exists {
+		used, err = service.deviceUsed(device)
+		if err != nil {
+			return
```

Comment:
Are You sure wanna swallow error here?
It's inconsistent with another return lines

Reply:
true, i don't

Reply:
actually code does not swallow it, but i will explicitly return with var for readability.